### DISCUSSION TITLE
大家使用低代码的场景在哪？

### DISCUSSION THREAD

**Discussion by xmsz** [taowen/awesome-lowcode#72] [General]
我们的话，现在用在

- 海报生成器，运营者自己在可视化画布拼接图像，并支持插入变量。类似于以前游戏开发中设计师的角色，就是操作编辑器，然后开发可以拿到代码
属于noCode

- cms，我们写好数据模型，用户可视化+表格操作数据。运营者操作起来就很方便
属于lowCode

- 一切其他可视化工具

但都只是实现可视化+转代码的阶段。

当然还有尝试过其他的，例如无代码开发网页/小程序，但很鸡肋，现在的效果跟20年前网站模板一样... 还有类似excle生成数据的无代码，但问题是这部分理论上应该有运营者操作，开发并不了解需求，但运营者学习成本又巨高

大家的场景都是哪些

**Comment by qmdx** [comment]
Java 低代码组件库  爱组搭 文档： http://aizuda.com

开源仓库 GitHub： https://github.com/aizuda  
Gitee ：  https://gitee.com/aizuda

欢迎交流！！

**Comment by LowCoderGitHub** [comment]
搞了个低代码行业的交流平台，大佬们有没有兴趣来新平台玩玩
https://www.lowcoder.com.cn/

**Comment by SelfDown** [comment]
我写的低代码https://github.com/SelfDown/collect-ui
